indico/modules/designer/templates/_list.html

```diff
@@ -18,6 +18,9 @@
             {% trans %}Print badges{% endtrans %}
         {%- endif -%}
     {%- endset %}
+    {% set extra_params %}
+        {% block render_badge_form_button_params scoped %} {% endblock %}
+    {% endset %}
```

Comment:
wouldn't a `template_hook` be better for this?

Reply:
It works the same, so I've changed it